From 6ab0f9664210091cd8fd7f8450e02b6358c59caa Mon Sep 17 00:00:00 2001
From: Faouaz TENOUTIT <faouaz.tenoutit@intel.com>
Date: Mon, 1 Sep 2014 09:15:53 +0200
Subject: [PATCH] Audio: Disable stack dump when doing DSP recovery.

Remove the stack & HW registers dump from the driver because the DSP
recovery is gracefuly managed (by audio driver & audio HAL).

Change-Id: Ief863f4b4e57dcbc2325b7ea66ccc59c6c30ed33
Signed-off-by: Faouaz TENOUTIT <faouaz.tenoutit@intel.com>
---
 sound/soc/intel/sst/sst_pvt.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sound/soc/intel/sst/sst_pvt.c b/sound/soc/intel/sst/sst_pvt.c
index 23fb319..d58fab0 100644
--- a/sound/soc/intel/sst/sst_pvt.c
+++ b/sound/soc/intel/sst/sst_pvt.c
@@ -372,8 +372,6 @@ void sst_do_recovery_mrfld(struct intel_sst_drv *sst)
 	sst->sst_state = SST_RECOVERY;
 	mutex_unlock(&sst->sst_lock);
 
-	dump_stack();
-	dump_sst_shim(sst);
 	cb_params.params = &reset_dapm;
 	cb_params.event = SST_PLATFORM_TRIGGER_RECOVERY;
 	reset_dapm = true;
-- 
2.1.0.rc1

